crates/ironclaw_webui_v2/static/styles/app.css

```
@@ -379,6 +379,8 @@ input:focus, select:focus, textarea:focus {
 .markdown-body {
   max-width: 100%;
   min-width: 0;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)